godot-macros/src/class/data_models/field_export.rs

```diff
@@ -40,6 +44,16 @@ pub enum ExportType {
     /// Can become other property hints, depends on context.
     Default,
 
+    /// ### GDScript annotations
+    /// - `@export_storage`
+    ///
+    /// ### Property hints
+    /// - `NONE`
+    ///
+    /// This is used to indicate that the property should be exported
+    /// but should not be visible in the editor.
+    Storage,
```

Comment:
While it doesn't have property _hints_, it has a property _usage_, could you add this?